### PR TITLE
Add Odomerter on 0x516

### PR DESCRIPTION
**Pull request by vanillagorillaa**
Car odometer in KM's

### CODE REVIEW

File: honda_civic_touring_2016_can.dbc [hunk 1]
```diff
@@ -305,6 +305,7 @@ BO_ 1039 XXX_25: 8 XXX
 BO_ 1108 XXX_26: 8 XXX
 
 BO_ 1302 XXX_27: 8 XXX
+SG_ ODOMETER : 7|24@0 + (1,0) [0,16777215] "Kilometers" NEO
```
Comment (rbiasini) [inline honda_civic_touring_2016_can.dbc]: there should be an indent space before SG. for the unit "km" is ok.

**Comment by vanillagorillaa** [comment]
Fixed

File: honda_civic_touring_2016_can.dbc [hunk 2]
```diff
@@ -305,6 +305,7 @@ BO_ 1039 XXX_25: 8 XXX
 BO_ 1108 XXX_26: 8 XXX
 
 BO_ 1302 XXX_27: 8 XXX
+ SG_ ODOMETER : 7|24@0 + (1,0) [0,16777215] "km" NEO
```
Comment (rbiasini) [inline honda_civic_touring_2016_can.dbc]: the "+" shouldn't have a space before

**Comment by vanillagorillaa** [comment]
Accidentally added line from different pull, should be fine now

File: honda_civic_touring_2016_can.dbc (hunk 2 above)
Comment (rbiasini) [reply honda_civic_touring_2016_can.dbc]: @vanillagorillaa , typo here still present

**Pull request merged**